src/Raven.Studio/typescript/models/resources/serverDashboard/machineResources.ts

```diff
@@ -24,22 +32,27 @@ class machineResources {
             const usage = this.processCpuUsage();
             return this.getCpuUsageClass(usage);
         });
-        
-        this.machineMemoryUsageClass = ko.pureComputed(() => {
-            const used = this.machineMemoryUsage();
+
+        this.commitedMemoryUsageClass = ko.pureComputed(() => {
+            const used = this.commitedMemory();
             return this.getMemoryUsageClass(used);
         });
 
         this.processMemoryUsageClass = ko.pureComputed(() => {
             const used = this.processMemoryUsage();
             return this.getMemoryUsageClass(used);
         });
+
+        this.processUsageTooltip = ko.pureComputed(() => {
+            return this.isLinux() ? "RavenDB Resident Memory" : "RavenDB Memory Usage";
```

Comment:
I think ti is better to get it from the server, no?

Reply:
I'm getting this from the server once

Reply:
I'm talking about the string you are using.